dsma-jce-lambda-example/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.fortanix.lambda</groupId>
+	<artifactId>dsma-jce-example</artifactId>
+	<version>1.0-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
+	<name>dsma-jce-example</name>
+	<!-- FIXME change it to the project's website -->
+	<url>http://www.example.com</url>
+
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.11</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.fortanix</groupId>
+			<artifactId>sdkms-jce-provider-dsma</artifactId>
+			<!-- change the version of this dependency as per installation -->
+			<version>4.22.2307</version>
```

Comment:
Change version to something like this `${dsma-jce.version}`

Reply:
Seems like a good feature but this doesn't seem to work with all of the maven versions ( ref: https://stackoverflow.com/questions/19123013/maven-version-with-a-property ).  But as a measure, I have indicated this version change step in README and POM file as well.

Reply:
I just meant version shouldn't be hard-coded. We can give it a common name & mention in readme to update it.

Reply:
Do you mean the project would still compile if i remove the hard coded version, since it will default to the version in the parent's pom.xml?

Reply:
Okay, i get what you mean, but i don't want to add an extra step for version specification just for compiling the project. It should compile and run with as few steps as possible as its a sample. The configuration page in the README talks about aws lambda configuration only, and I think it keeps the example quicker to deploy and test.